web/src/games/zooparade/components/bbuttons.stories.tsx

```diff
@@ -1,7 +1,9 @@
 import { BButtons } from './bbuttons';
+import { gameDecorator } from '../decorators/game';
```

Comment:
What does that do?

Reply:
It is a decorator for Story that wraps GameProvider and ReduxProvider around the Story to allow translated text inside of it.